Topics/Development_Process/Github_Pages.md

```diff
@@ -0,0 +1,91 @@
+# GitHub Pages
+
+If you have ever created a "site" before using HTML, maybe with CSS and perhaps Javascript, and wondered how you could host it online through a link, here's your solution.
+
+From the [GitHub docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages):
+
+> GitHub Pages is a static site hosting service that takes HTML, CSS, and JavaScript files straight from a repository on GitHub, optionally runs the files through a build process, and publishes a website.
+
+In other words, it is a **free static website host.** It's reliable, your pages will remain up forever (unless GitHub changes this), and can handle a reasonable amount of bandwidth (100GB/mo), given you aren't serving images or large files.
+
+**Note:** Free for public repositories. You need GitHub pro if you want to do this for private repositories. The deployed "page" will be public but your repo will remain private. Students should be able to get Pro for free.
+
+## Caveats
+
+Many people use GitHub pages to host blogs and personal portfolios. You can also host web applications, although I would only suggest using it for lightweight stuff -- mainly, things that would be **read**, not **written.** In other words, I **would not** use GitHub pages as a substitute for Google Forms, even though you could still do that. **I would also not use GitHub pages if you need to pair a frontend with a backend.** There are other great alternatives that you can find in this wiki, such as [Vercel](https://vercel.com/), [Render](https://render.com/), and so on.
```

Comment:
It might be good to go more in-depth on why Github pages is not a good option for these use cases. For example, it may be helpful to explain that Github pages can't run a server. As a result, any server-side functionality such as authentication/authorization will not work.